### PR TITLE
Rework README + featuring Tawasal

### PR DESCRIPTION
**Pull request by TimLariviere**

### CODE REVIEW

**Comment by TimLariviere** [comment]
@SergejDK Does that look ok to you?

File: docs/Fabulous.XamarinForms/they-use-fabulous.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+### Tawasal
+
+<img alt="Tawasal" src="../assets/apps-using-fabulous/tawasal/logo.png" height="75" />
+
+[Website](https://www.tawasal.ae) | [Google Play Store](https://play.google.com/store/apps/details?id=com.tawasul.messenger) | [App Store](https://apps.apple.com/app/id1507761438)
+
+**Description**
+
+Tawasal is a secure multi-purpose messenger and superapp, offering free voice, text, videoconferencing and lifestyle services to clients around the world. Tawasal introduces a full suite of exciting features with something for everyone, from live sports text broadcasts and giveaways, to payments and news services.
+
```
Comment (SergejDK) [inline docs/Fabulous.XamarinForms/they-use-fabulous.md:16]: `live sports, text, broadcasts` just adding some comma to make it clearer

File: docs/Fabulous.XamarinForms/they-use-fabulous.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+**Why they chose Fabulous?**
+
+Tawasal team was looking for cross platform solution that was stable & supported. When we started looking into what Xamarin has to offer, we came across Fabulous - and due to usage of modern & reactive MVU pattern, as well as the ability to build basically anything using native Xamarin.iOS / Xamarin.Android bindings (using less code & functional approach with F#), while still maintaining a large shared codebase our choice was surely Fabulous.
```
Comment (SergejDK) [inline docs/Fabulous.XamarinForms/they-use-fabulous.md:19]: `Our team at Tawasal was looking for a cross platform solution that was stable & supported.`

The first sentence is written different than the other part.

**Review by SergejDK — COMMENTED** [review]
All in all it looks pretty good.

File: README.md [hunk 3]
```diff
@@ -2,46 +2,34 @@
...
+ [![Build Status](https://dev.azure.com/timothelariviere/Fabulous/_apis/build/status/Full%20Build?branchName=master)](https://dev.azure.com/timothelariviere/Fabulous/_build/latest?definitionId=7&branchName=master) [![Fabulous.XamarinForms NuGet version](https://badge.fury.io/nu/Fabulous.XamarinForms.svg)](https://badge.fury.io/nu/Fabulous.XamarinForms) [![Join the chat at https://gitter.im/fsprojects/Fabulous](https://badges.gitter.im/fsprojects/Fabulous.svg)](https://gitter.im/fsprojects/Fabulous?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
 Never write a ViewModel class again! Conquer the world with clean dynamic UIs!
 
-* [Documentation](https://fsprojects.github.io/Fabulous/)
+Fabulous allows you to combine the power of functional programming and the ultra-simple Model-View-Update architecture to build any kind of mobile and desktop applications with an expressive, dynamic and clean UI DSL. Go cross-platform with Fabulous for Xamarin.Forms and target iOS, Android, Mac, WPF and more!
```
[the remark this answers is not in the document]
Comment (TimLariviere) [reply README.md]: I used `simple` like you suggested

**Pull request merged**